addr.go

```diff
@@ -34,6 +41,30 @@ func ParseAddr(s string) (*Addr, error) {
 	return &Addr{IPNet: m, Label: label}, nil
 }
 
+// ParseAddr parses the string representation of an address in the form $ip/$netmask.
+func ParseAddr(s string) (*Addr, error) {
+	m, err := ParseIPNet(s)
+	if err != nil {
+		return nil, err
+	}
+	return &Addr{IPNet: m}, nil
+}
```

Comment:
When you ricive data from other source (like rpc call, shell input etc...), you want to check if the network address is correct and you dont want to use net.ParseIP followed with netlinka.Addr{...}, or netlink.ParseAddr. I just want to keep all network stuff in netlink package, this was the purpose of spliting this into two functions, what do you think about this ?